binder/jupyterlab-workspace.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
The workspace isn't currently coming up, which is fine. I recommend that we remove this file for now.